pyproject.toml

```diff
@@ -0,0 +1,75 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "instana"
+dynamic = [
+    "version",
+]
+description = "Python Distributed Tracing & Metrics Sensor for Instana."
+readme = "README.md"
+requires-python = ">=3.7"
+license = "MIT"
+keywords = [
+    "performance",
+    "opentracing",
+    "metrics",
+    "monitoring",
+    "tracing",
+    "distributed-tracing",
+]
+authors = [
+    { name = "Instana Team Python Engineers", email = "info@instana.com" },
```

Comment:
If `info@instana.com` still works, then its days should be numbered. We shouldn't use it anymore.

Reply:
Got it. What do you think about the authors list @Ferenc-?

Reply:
As the [specification](https://packaging.python.org/en/latest/specifications/pyproject-toml/#authors-maintainers) states that only the name is a valid entry, I removed the email address.